Bottom: 88d8529909a75dee3e6590e8fae9dd979b208d9c
Top:    b1e182d0db739bc883fed5a6fbf1cdd6d6feeb75
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-12 01:31:18 +0000

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index f3bb2cb..64ff374 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1190,6 +1190,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1212,6 +1213,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
